### PR TITLE
{Monitor} `monitor autoscale update`: Support `--count` equal `0`

### PR DESCRIPTION
**Pull request by kairu-ms**
**Related command**

**Description**

Fix ICM: https://portal.microsofticm.com/imp/v3/incidents/details/311436712/home

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Monitor

File: src/azure-cli/azure/cli/command_modules/monitor/operations/autoscale_settings.py [hunk 1]
```diff
@@ -72,8 +72,7 @@ def autoscale_update(instance, count=None, min_count=None, max_count=None, tags=
     if enabled is not None:
         instance.enabled = enabled
 
-    if any([count, min_count, max_count]):
-
+    if count is not None or min_count is not None or max_count is not None:
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/operations/autoscale_settings.py:75]: `count == 0` now is `true` (previous it is `false`)?

Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/operations/autoscale_settings.py:75]: `count == 0` now is `true` (previous it is `false`)?

Comment (kairu-ms) [reply src/azure-cli/azure/cli/command_modules/monitor/operations/autoscale_settings.py:75]: Yes

Comment (kairu-ms) [reply src/azure-cli/azure/cli/command_modules/monitor/operations/autoscale_settings.py:75]: When count == 0 any([count, min_count, max_count]) will response false

**Review by jsntcy — APPROVED** [review]

**Pull request merged**